### ISSUE TITLE
FreeDOS :-)

### ISSUE THREAD

**Issue by evert-mouw** [Manawyrm/nvme-int13h-optionrom#1]
Pretty cool! Not sure if I understand it correctly, but in theory, one could boot FreeDOS?

I guess that would be more useful with write support -- some option to switch it on would be nice.

**Comment by Manawyrm** [comment]
> but in theory, one could boot FreeDOS?

Yep, haven't tested it, but as MS-DOS is working well, I'm guessing FreeDOS will just work fine.

> more useful with write support

Indeed! I also don't want people breaking their filesystems by accident :)
I think I'll write some CI/integration stuff soon-ish, so people can try this easier and I might then have a look at write support.
Writing is always pretty dangerous and needs a lot of sanity checks, as I really don't want to mess up people's NVMe drives by accident.

**Comment by evert-mouw** [comment]
DOS never dies 🧙 
I'll bookmark your project.
